packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -78,6 +79,10 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 
 ## Important Fixes
 
+- **POS 10.5.0**:
+
+  - Added support for a product variant id field in the \`ProductApi\` interface.
```

Comment:
We actually need to include this here instead since it's a [relevant change to API Version](https://github.com/Shopify/ui-extensions/blob/d1ae3265a45290f262b34d06302ba2f766398e4b/packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts#L56)